bot/seasons/evergreen/bookmark.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import random
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours, ERROR_REPLIES
+
+log = logging.getLogger(__name__)
+
+
+class Bookmark(commands.Cog):
+    """A cog that creates personal bookmarks by relaying a message to the user's DMs."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(name="bookmark", aliases=("bm", "pin"))
+    async def bookmark(self, ctx: commands.Context, target_message: discord.Message, *,
+                       title: str = None) -> None:
+        """Send you a link to the provided message in DM."""
+        log.info(f"{ctx.author} bookmarked {target_message.jump_url} with hints {title}.")

```

Comment:
Since the variable has been fully renamed
```suggestion
        log.info(f"{ctx.author} bookmarked {target_message.jump_url} with title {title}.")
```